manifests/dev/vsphere-67u3/vanilla/vsphere-csi-driver.yaml

```diff
@@ -89,7 +89,6 @@ apiVersion: v1
 data:
   "csi-migration": "false"
   "csi-auth-check": "true"
-  "online-volume-extend": "false"
```

Comment:
6.7U3 CSI YAML does not even have a resizer sidecar. Our driver does not even publish the resize capability for this version. Therefore, it is not possible that CSI gets a call to ControllerExpandVolume. That is why I have not mentioned the FSS here. It is by default False but I prefer not mentioning it here to avoid confusing customers.

Reply:
Does it mean the k8s admin has need to apply a different manifest after vSphere is upgraded to 7.0U2? If yes, that does not sound good.

Reply:
Marked it as true with a comment to upgrade the driver